examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -665,7 +665,7 @@ void WindowManager::UpdateLEDs()
 #if CHIP_ENABLE_OPENTHREAD
             if (mState.isThreadProvisioned && mState.isThreadEnabled)
 #else
-            if (mState.isWiFiProvisioned && mState.isWiFiEnabled)
+            if (ConnectivityMgr().IsWiFiStationProvisioned() && ConnectivityMgr().IsWiFiStationEnabled())
```

Comment:
Why not keep variables and set them correctly. This should be done for all apps in base applications if I am not mistaken

Reply:
Earlier, in common WindowApp.cpp, these variables are being set in 'Run()' API. This API is missing in WindowManager.cpp after refactoring. Since there are only 1-2 instances of these variables, we have removed them and directly added APIs.

Reply:
If not used at all, I would expect the variables to be deleted all together our `WindowManager.h`
Also calling the following apis `ConnectivityMgr().IsWiFiStationProvisioned() and ConnectivityMgr().IsWiFiStationEnabled()` might work for wifi, But it I problematic to call it frequently for thread ( slows the system a lot due to nvm3 reads for the dataset)

BaseApplication.cpp already deals with checking the provision state in a safe way. I would leverage that instead.

Lastly, this whole UpdateLEDs() function in the window-app seems to counter what is done in the `baseApplication` `LightTimerEventHandler`. That is definitely and issue.

The rework of this example might have missed some key elements. We have to fix this correctly so it behaves like all our examples

Reply:
Thanks for the comments.

**LightTimerEventHandler** function in BaseApplication.cpp can be used to handle Status LED (LED0), UpdateLEDs() function will be needed for window-app to deal with ActionLED (LED 1). Removed the code which handles LED0 in UpdateLEDS() function .